app/liquid_tags/loom_tag.rb

```diff
@@ -20,7 +20,8 @@ def render(_context)
   end
 
   def extract_video_id(input)
-    match = pattern_match_for(input, [REGISTRY_REGEXP])
+    input_params_removed = input.split("?")[0] # Loom URLs with params fail valid-URL check
```

Comment:
I like this a lot. Cleaning up the inputs when feasible is often better than raising a pedantic error. `git` does the pedantic thing, even when it knows what you want:

```
➜  forem git:(msarit/better-error-msg-invalid-embed-links) git di main... app -w
fatal: option '-w' must come before non-option arguments
```

The way you're doing it provides a better experience. 💯